_layouts/base.html.slim

```diff
@@ -93,6 +93,12 @@ body.antialiased
       - unless page.url.start_with? '/news/'
         .large-4.columns.secondary
           - if page.url =~ /^\/docs\/.+/
+            - if page.toc
```

Comment:
The correct way to generate a toc is to delegate to the outline node handler. You can see an example here:

https://github.com/asciidoctor/asciidoctor-backends/blob/master/slim/html5/block_toc.html.slim